packages/plugins/image/src/ui/Placeholder.tsx

```diff
@@ -32,6 +32,7 @@ const Placeholder = ({ attributes, children, blockId }) => {
         className={`yoopta-button yoo-image-p-[12px_36px_12px_12px] yoo-image-flex yoo-image-items-center yoo-image-text-left yoo-image-w-full yoo-image-overflow-hidden yoo-image-rounded-[3px] yoo-image-text-[14px] yoo-image-text-[rgba(55,53,47,0.65)] yoo-image-relative yoo-image-cursor-pointer yoo-image-border-none yoo-image-bg-[#efefef] yoo-image-transition-[background-color_100ms_ease-in] hover:yoo-image-bg-[#e3e3e3]`}
         onClick={() => setIsUploaderOpen(true)}
         disabled={loading}
+        type="button"
```

Comment:
Thanks for this fix :) 
Can you please add this attribute to another Placeholder components (in Embed, Video, File)?

Reply:
Hey @Darginec05 yeah for sure 👍

Reply:
thanks

Reply:
@Darginec05 just updated the PR

Reply:
super!